openapi3/schema_formats.go

```diff
@@ -10,15 +11,86 @@ const (
 	FormatOfStringForUUIDOfRFC4122 = `^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$`
 )
 
-var SchemaStringFormats = make(map[string]*regexp.Regexp, 8)
+// FormatType is the type of format validation used
+type FormatType string
 
+// FormatTypeRe is regexp based validation
+const FormatTypeRe = "re"
+
+// FormatTypeCallback is callback based validation
+const FormatTypeCallback = "callback"
+
+//FormatCallback custom check on exotic formats
+type FormatCallback func(Val string) error
+
+//Format is the format type context fo validate the format
```

Comment:
try harder

Reply:
you mean on the comments? suggestions?

Reply:
Yes I'm not sure what you're trying to convey here. Just delete it?